space/space.go

```diff
@@ -168,6 +170,10 @@ func (r *GormRepository) listSpaceFromDB(ctx context.Context, start *int, limit
 		db = db.Limit(*limit)
 	}
 	db = db.Select("count(*) over () as cnt2 , *")
+	if q != nil {
+		db = db.Where("LOWER(name) LIKE ?", "%"+ strings.ToLower(*q) + "%")
```

Comment:
I'm using parameterized queries. The concatenation is only to enclose the search text in `%`, but the value (even if tainted) is passed to the SQL query using a parameterized query (which should remove the taint).